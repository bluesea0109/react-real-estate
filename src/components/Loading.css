.ui.loading.segment:after {
  position: absolute;
  content: '';
  top: 39%;
  left: 50%;
  margin: -3.5em 0px 0 -3.5em;
  width: 7em;
  height: 7em;
  animation: segment-spin 0.6s linear;
  animation-iteration-count: infinite;
  border-radius: 500rem;
  border-color: #767676 rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.1);
  border-style: solid;
  border-width: 0.2em;
  box-shadow: 0 0 0 1px transparent;
  visibility: visible;
  z-index: 101;
}
