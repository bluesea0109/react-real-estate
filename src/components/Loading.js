import React from 'react';

import { Dimmer, Loader } from './Base';

const Loading = () => (
  <Dimmer active inverted>
    <Loader>Loading...</Loader>
  </Dimmer>
);

export default Loading;
