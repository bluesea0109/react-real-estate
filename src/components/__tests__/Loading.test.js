import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../Loading';
import 'jest-styled-components';

it('renders correctly', () => {
  const tree = renderer.create(<Loading />).toJSON();
  expect(tree).toMatchSnapshot();
});
