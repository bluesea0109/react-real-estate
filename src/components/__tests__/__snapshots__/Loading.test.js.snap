// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ui basic segment "
  style={
    Object {
      "margin": "6em auto",
    }
  }
>
  <div
    className="ui centered card "
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "#f9f8f7",
        "boxShadow": "none",
        "minWidth": "380px",
      }
    }
  >
    <img
      className="ui tiny centered image "
      src="loading.gif"
      style={
        Object {
          "background": "unset",
          "marginTop": "1em",
        }
      }
    />
    <div
      className="content"
      style={
        Object {
          "borderTop": "none",
        }
      }
    >
      <h2
        className="ui center aligned header"
      >
        <div
          className="content"
        >
          Loading, please wait...
        </div>
      </h2>
    </div>
  </div>
</div>
`;
